KsApi/mutations/PostCommentMutation.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public struct PostCommentMutation<T: GraphMutationInput>: GraphMutation {
+  var input: T
+
+  public init(input: T) {
+    self.input = input
+  }
+
+  public var description: String {
+    return """
+    mutation ($input: PostCommentInput!) {
```

Comment:
More just for my own knowledge, but I guess we don't use the DSL (GraphSchema) for mutations?

Reply:
I think @justinswart has more historical context here but based on the code, we are not "building" the mutation but instead relying on that `description` property in our Service helper.